### ISSUE TITLE
[Example] Mouse example for the `Up` and `Down` functions

### ISSUE THREAD

**Issue by sbplat** [sbplat/Macro-API#5]
Create an example using the `Up` and `Down` mouse functions and place it in the `examples` folder.

Function declarations for `Up` and `Down` (see `include/macro/mouse.h`):
```c++
void Down(Button button);
void Up(Button button);
```

**Comment by unclebinary1001** [comment]
@sbplat Can I take this up?

**Comment by sbplat** [comment]
I can assign this to you after you finish #3.

**Comment by LegendaryGene** [comment]
Can I work on this Issue if this is still available? If yes, could you just elaborate what sort of example I have to make?

**Comment by sbplat** [comment]
> Can I work on this Issue if this is still available? If yes, could you just elaborate what sort of example I have to make?

Sure! I've assigned this issue to you.
You may refer to #3 and #31 as examples.

**Comment by LegendaryGene** [comment]
Hey sorry for the delay, got a bit sidetracked. Will start work on this in a bit, to confirm I just have to create an example using the Up and Down mouse functions (and other mouse functions as required) and check if it is correctly working right?

**Comment by sbplat** [comment]
Yes, that's correct. Try to be as creative as possible with your example!

**Comment by sbplat** [comment]
Hi, I'm going to unassign this issue from you since you haven't worked on it recently.
